Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -147,37 +161,468 @@ bool loadCanceled
         TaskCreationOptions.None, LargeStackScheduler).Unwrap();
     }
 
+    // Called only in the case there is a parsing or resolution error on the document
+    public void PublishVerificationDiagnostics(DafnyDocument document) {
```

Comment:
Why not inline this method?

Reply:
Because it is used in two places in TextDocumentLoader.

Reply:
But it only contains a single call right? `diagnosticPublisher.PublishVerificationDiagnostics(document);` What's the use of wrapping that in another method?

Reply:
Ok, I inlined this method.